pageserver/src/tenant/storage_layer/delta_layer.rs

```diff
@@ -475,7 +483,8 @@ impl DeltaLayer {
     /// Open the underlying file and read the metadata into memory, if it's
     /// not loaded already.
     ///
-    fn load(&self) -> Result<RwLockReadGuard<DeltaLayerInner>> {
+    fn load(&self, ctx: &RequestContext) -> Result<RwLockReadGuard<DeltaLayerInner>> {
+        self.access_stats.record_access(todo!(), ctx);
```

Comment:
Should probably do this only in the case that this thread loads (review right hand line 500)

Reply:
Nevermind, this is called through all interesting parts (before iters, get_reconstruct_data) and we are not interested in the actual load, but the access. Looking into how it's ensured that it happens before iters, but resolving for now.